pkgs/stdenv/darwin/default.nix

```diff
@@ -278,8 +278,8 @@ in rec {
     # enables patchShebangs above. Unfortunately, patchShebangs ignores our $SHELL setting
     # and instead goes by $PATH, which happens to contain bootstrapTools. So it goes and
     # patches our shebangs back to point at bootstrapTools. This makes sure bash comes first.
```

Comment:
Edit this comment to match the new reality, too? I also feel like if we can use your `disallowRequisites` patch with this, we can skip `pkgs.bash` in either list, which would be best. I'd be very down to collaborate getting all the stdenv deps to use `strictDeps` :).

Reply:
Yeah that sounds good! It might have to go in another PR for now - I'm more worried about fixing cross compilation right now.

One thing these are all going to need though is bash in the buildInputs. I think there's quite a few of those that don't have it right now.

I think we could also do an override here that does something like:

```
stdenv = super.stdenv // { mkDerivation = args: super.stdenv.mkDerivation (args // { strictDeps = true; }); };
```

To avoid individually setting all of these.

Reply:
Well I do want to audit those packages. I think I've decided this PR and the `disallowRequites` one are good as is, and then all the per-package cleanup can happen in a 3rd PR.